src/autobot.ts

```diff
@@ -1,20 +1,16 @@
-import {
-  SyncBailHook,
-  SyncHook,
-  AsyncSeriesWaterfallHook,
-  AsyncSeriesHook,
-  AsyncParallelBailHook,
-  AsyncParallelHook,
-} from "tapable";
+import { SyncBailHook, SyncHook, AsyncSeriesWaterfallHook, AsyncSeriesHook } from "tapable";
 import { Application, Context } from "probot";
 import { WebhookPayloadPullRequest } from "@octokit/webhooks";
 import { fetchConfig, Config } from "./config";
-import { Plugin, AppPlugin, PullRequestPlugin, UninitializedPlugin } from "./plugin";
+import { Plugin, AppPlugin, PullRequestPlugin, UninstantiatedPlugin } from "./plugin";
 import { ReposCreateStatusParams } from "@octokit/rest";
 import to from "await-to-js";
 import { getLogger } from "./utils/logger";
+import { isProduction } from "./utils/env";
+import { fromPairs } from "lodash";
 
 const logger = getLogger("autobot");
+const STATUS_CONTEXT = isProduction ? "auto-bot" : "auto-bot-dev";
```

Comment:
This is so that people can test the dev version without it conflicting with the prod one. Basically can have two auto apps installed at the same time.